CriteoPublisherSdk/Tests/Utility/Categories/NSUserDefaults+GDPR.m

```diff
@@ -27,6 +27,11 @@ - (void)clearGdpr {
   [self removeObjectForKey:NSString.gdprAppliesUserDefaultsKeyTcf1_1];
   [self removeObjectForKey:NSString.gdprConsentStringUserDefaultsKeyTcf1_1];
   [self removeObjectForKey:NSString.gdprPurposeConsentsStringForTcf2_0];
+  for (int id = 0; id <= 10; ++id) {
```

Comment:
Where does this 10 comes from ? What about adding a comment or extracting it in a named constant ?

Reply:
Done